### DISCUSSION TITLE
Release v1.0.22 / 5.55.22

### DISCUSSION THREAD

**Discussion by DavidXanatos** [sandboxie-plus/Sandboxie#1858] [Announcements]
![0 5 5](https://user-images.githubusercontent.com/3890945/105014988-575cb680-5a41-11eb-90d5-d942d28184b2.png)

This build brings a few fixes as well as a few small features, most notably support for the new windows 11 explorer context menu.
For now to get it enabled the setting needs to be manually toggled on windows 11, a later installer update will automate this for win 11.

#### You can support the project through donations, any help will be greatly appreciated.

[<img src="https://www.paypalobjects.com/en_US/AT/i/btn/btn_donateCC_LG.gif">](https://xanasoft.com/go.php?to=donate)

# ChangeLog

### Added
- added auto update download and silent install option to sandman.exe [#917](https://github.com/sandboxie-plus/Sandboxie/issues/917)
- trace monitor mode can now also save to file [#1851](https://github.com/sandboxie-plus/Sandboxie/issues/1851)
- trace log now shows ipc object type information
- added support for windows 11 context menus

### Fixed
- fixed sandman crash issue [#1846](https://github.com/sandboxie-plus/Sandboxie/issues/1846)
- fixed issue with windows server 2022 build 20348
- fixed translation switching issues [#1852](https://github.com/sandboxie-plus/Sandboxie/issues/1852)

<hr /><em>This discussion was created from the release <a href='https://github.com/sandboxie-plus/Sandboxie/releases/tag/1.0.22'>Release v1.0.22 / 5.55.22</a>.</em>

**Comment by darkred** [comment]
> support for the new windows 11 explorer context menu.
> For now to get it enabled the setting needs to be manually toggled on windows 11, a later installer update will automate this for win 11.

How to enable/manually toggle this setting ?
There's no relevant new entry in Options | Global Settings | Shell integration :
<details>
<summary>screenshot</summary>

![](https://user-images.githubusercontent.com/723651/168484015-55c676af-41bd-43b8-8fa4-f458e5bbd4c5.jpg)
</details>

I've tried both updating and fresh install of 1.0.22 (win 11 x64) but it's the same.

Thank you

**Reply by DavidXanatos** [reply]
You use the existing setting uncheck apply check apply done

**Reply by darkred** [reply]
Ok, so I unchecked the "Add 'Run Sandboxed' to the explorer context menu" setting but unfortunately no new entry is added to windows context menu.

**Reply by DavidXanatos** [reply]
try restarting explorer

**Reply by darkred** [reply]
I tried that, and also tried restarting windows, but unfortunately it doesn't make a difference.

**Comment by darkred** [comment]
I also noticed that, after I uncheck it (+Apply), and try to check it again, then the setting's checked stated is not stored (i.e. it remains unchecked), neither with Ok nor with Apply.

**Reply by DavidXanatos** [reply]
that looks as if the installation of the msix fails, what exact windows 11 build do you have?

**Reply by darkred** [reply]
It's version 21H2 build 22000.675 (win 11 stable, fully updated).

<details>
<summary>Screen capture of the stuck setting:</summary>

https://user-images.githubusercontent.com/723651/168601320-ba7c92ed-0963-4513-a9d6-0ab0bdfe23d6.mp4
</details>

**Comment by abet00** [comment]
Hello, im experiencing issues with latest sandboxie version on steam, i cannot open dota2 exe

error code:
CappSystemdict::LoadSystemAndDependencies():
AppSystemDict:: Error in Connect() of interface 'RenderDeviceMgr001'!
![1231](https://user-images.githubusercontent.com/107626325/174072112-f14915fe-4570-436c-b1a8-93f9314d99fa.jpg)

Windows 11 PRO
Version 21H2 (OS build 22000.739)
